src/modules/dashboard/checks.ts

```diff
@@ -122,3 +122,12 @@ export const canFinalizeTask = (task: AnyTask, roles: ROLES[]) =>
   canFund(roles);
 
 export const canRecoverColony = hasRoot;
+
+/*
+ * Suggestions
+ */
+
+export const hasUpvotedSuggestion = (
```

Comment:
So I think we should actually change the logic behind your own upvotes.

![Screenshot from 2020-01-22 15-30-11](https://user-images.githubusercontent.com/1193222/72898088-1ba6e800-3d2c-11ea-832e-006fdc1031fc.png)

We have two ways to go about it:
- Either you upvote your submission automatically when you create it
- Either you can't upvote your own suggestion (reddit style)

This comes down to what is the number of upvotes we want the suggestion to start with: 0 (you can't upvote yours) or 1 (you automatically upvote)

What are your thoughts on this ?

Maybe even @JoinColony/product can chime in ....

Reply:
Hmmm interesting ideas....

> Either you can't upvote your own suggestion (reddit style)

I think this makes the most sense to me. But once again, I'm curious about @JoinColony/product's input here.

Great call, @rdig

Reply:
This is not how Reddit works. Reddit is the opposite: you automatically upvote your own post, and have the option of removing it or even down voting it. (I just tested this.)

I prefer that: automatically add a +1 from the person who posted, and allow them to remove if they want.

Reply:
Great, thanks guys!

Reply:
Update: solved via https://github.com/JoinColony/colonyServer/pull/37